GCMessengerSDKSample/Podfile.lock

```diff
@@ -0,0 +1,80 @@
+PODS:
+  - GenesysCloud (4.x.y-dev):
+    - GenesysCloud/Global (= 4.x.y-dev)
+  - GenesysCloud/Bot (4.x.y-dev):
+    - GenesysCloud/Core
+    - GenesysCloud/Search
+    - GenesysCloudBot
+  - GenesysCloud/Core (4.x.y-dev):
+    - GenesysCloudAccessibility
+  - GenesysCloud/Global (4.x.y-dev):
+    - GenesysCloud/Bot
+    - GenesysCloud/Core
+    - GenesysCloud/Live
+    - GenesysCloud/Meesenger
+    - GenesysCloud/Search
+  - GenesysCloud/Live (4.x.y-dev):
+    - GenesysCloud/Core
+    - GenesysCloudBold
+  - GenesysCloud/Meesenger (4.x.y-dev):
+    - GenesysCloudMessenger
+  - GenesysCloud/Search (4.x.y-dev):
+    - GenesysCloud/Core
+  - GenesysCloud/UnitTests (4.x.y-dev)
+  - GenesysCloudAccessibility (1.x.y-dev)
+  - GenesysCloudBold (1.x.y-dev):
+    - GenesysCloudCore
+  - GenesysCloudBot (1.x.y-dev):
+    - GenesysCloudCore
+  - GenesysCloudCore (1.x.y-dev)
+  - GenesysCloudCore/UnitTests (1.x.y-dev)
+  - GenesysCloudMessenger (1.x.y-dev):
+    - GenesysCloudCore
+    - GenesysCloudMessengerTransport
+  - GenesysCloudMessenger/UnitTests (1.x.y-dev):
+    - GenesysCloudCore
+    - GenesysCloudMessengerTransport
+  - GenesysCloudMessengerTransport (2.3.1)
+
+DEPENDENCIES:
+  - GenesysCloud (from `../../../BitBucket/mobiledx-ios-sdk-ui/GenesysCloud-Dev.podspec`)
+  - GenesysCloud/UnitTests (from `../../../BitBucket/mobiledx-ios-sdk-ui/GenesysCloud-Dev.podspec`)
+  - GenesysCloudAccessibility (from `../../../BitBucket/mobiledx-ios-sdk-accessibility/GenesysCloudAccessibility-Dev.podspec`)
+  - GenesysCloudBold (from `../../../BitBucket/mobiledx-ios-sdk-live-engine/GenesysCloudBold-Dev.podspec`)
+  - GenesysCloudBot (from `../../../BitBucket/mobiledx-ios-sdk-bot-engine/GenesysCloudBot-Dev.podspec`)
+  - GenesysCloudCore (from `../../../BitBucket/mobiledx-ios-sdk-core/GenesysCloudCore-Dev.podspec`)
+  - GenesysCloudCore/UnitTests (from `../../../BitBucket/mobiledx-ios-sdk-core/GenesysCloudCore-Dev.podspec`)
+  - GenesysCloudMessenger (from `../../../BitBucket/mobiledx-ios-sdk-messenger/GenesysCloudMessenger-Dev.podspec`)
+  - GenesysCloudMessenger/UnitTests (from `../../../BitBucket/mobiledx-ios-sdk-messenger/GenesysCloudMessenger-Dev.podspec`)
+  - GenesysCloudMessengerTransport (= 2.3.1)
```

Comment:
Why the sample needs to declare of the transport dependency ? it should be internal.
Is it due to the issue with the 2.3.1 version?

Reply:
You are right, it happened because the last test I did was on the local frameworks, I've pushed a fix for that